content/yaml-publishing/huawei-appgallery.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Huawei AppGallery
+description: How to deploy an app to Huawei AppGallery using codemagic.yaml
+weight: 4
+---
+
+Codemagic enables you to automatically publish your iOS or Android app to Huawei AppGallery.
+
+## Creating an API Client
+An API client is an identity credential used by AppGallery Connect to manage user access to AppGallery Connect APIs. Before accessing an API, you must create an API client with the permission for accessing the API. The procedure is as follows:
+
+1. Sign in to [AppGallery Connect](https://developer.huawei.com/consumer/en/service/josp/agc/index.html) and click **Users and permissions**.
+2. Go to **API key** > **Connect API** and click **Create**.
+3. For **Name**, enter a custom client name. Set **Project** to **N/A**, select the required roles, and click **OK**.
+
+{{<notebox>}}
+NOTE: Set Project to N/A to define the API client as a team-level one. Otherwise, the result code 403 will be returned during API calls.
+{{</notebox>}}
+
+4. After the client is successfully created, you need to save the **Client ID** and the **Client Secret** (**Key**) to [environment variables](/variables/environment-variable-groups/#storing-sensitive-valuesfiles) in a group named **app_gallery** for example.
+
+## Get the App ID
+1. Sign in to [AppGallery Connect](https://developer.huawei.com/consumer/en/service/josp/agc/index.html) and click **My apps** and select you app.
+3. Go to **App Information** and save the **App ID** to the environment variables as well in the **app_gallery** environment group.
+
+## App Signing
+If you want to release your app in App Bundle format and use the dynamic delivery feature, this service is required.
```

Comment:
This sentence is ambiguous. Do you mean that the app will have to be signed if you want to release the app?

Reply:
Yes, when publishing an `aab`.